report/report.go

```diff
@@ -21,15 +21,18 @@ import (
 	"github.com/ajeddeloh/vcontext/tree"
 )
 
+// EntryKind represents an Entry's severity
```

Comment:
Minor: docstrings in this patchset have inconsistent final dot.

Reply:
Fixed